go/vt/mysqlctl/tmutils/schema.go

```diff
@@ -273,9 +275,7 @@ func DiffSchema(leftName string, left *tabletmanagerdatapb.SchemaDefinition, rig
 	for leftIndex < len(left.TableDefinitions) && rightIndex < len(right.TableDefinitions) {
 		// extra table on the left side
 		if left.TableDefinitions[leftIndex].Name < right.TableDefinitions[rightIndex].Name {
-			if schema.IsInternalOperationTableName(left.TableDefinitions[leftIndex].Name) {
-				log.Infof("found internal table %v, skipping in schema diff", left.TableDefinitions[leftIndex].Name)
```

Comment:
The reason we had added these logs were for visibility in case some table mysteriously failed to get resharded and the user could identify why from the logs. But removing it should be fine.

Reply:
Yeah, that was my thinking as well too (I added these specific log messages here). But it ends up as a lot of log messages in total and makes it harder to find the signal in the noise. I feel comfortable removing them as I double checked the REGEXPs and they are very precise:
  - https://github.com/vitessio/vitess/blob/a033d76e25a1ce5332b3956f8db9801f07bccb81/go/vt/schema/tablegc.go#L47
  - https://github.com/vitessio/vitess/blob/a033d76e25a1ce5332b3956f8db9801f07bccb81/go/vt/schema/online_ddl.go#L39